vault/resource_consul_secret_backend.go

```diff
@@ -288,3 +296,20 @@ func consulSecretBackendExists(d *schema.ResourceData, meta interface{}) (bool,
 func consulSecretBackendConfigPath(backend string) string {
 	return strings.Trim(backend, "/") + "/config/access"
 }
+
+func consulSecretsBackendCustomizeDiff(_ context.Context, diff *schema.ResourceDiff, _ interface{}) error {
+	newToken := diff.Get("token").(string)
+	newBootstrap := diff.Get("bootstrap").(bool)
+
+	// If the user sets bootstrap to false but doesn't provide a token, disallow it.
+	if newToken == "" && !newBootstrap {
+		return fmt.Errorf("field `bootstrap` must be set to true when `token` is unspecified")
```

Comment:
best to use single quotes over back ticks,  shell interprets those as subshells.